demo/specs/quickstart/README.md

```diff
@@ -15,6 +17,7 @@ kubectl get pod -A
 
 #### Deploy the 4 example apps discussed in the slides
 ```console
+cd v1
```

Comment:
Can you make it its own step to decide which version directory to cd into, and then show the command to cd into `v1` as an example of that.

Reply:
done